jenkins-jobs/jobs/EclipsePluginsJob.groovy

```diff
@@ -0,0 +1,95 @@
+setupReleaseBuild('release')
```

Comment:
But this is doing a setup action here. It's setting up the release build steps in the job.